src/main/java/Duke/Parser/Parser.java

```diff
@@ -0,0 +1,59 @@
+package Duke.Parser;
+
+import Duke.Command.*;
```

Comment:
Avoid wildcard imports. It might be better to explicitly import the packages you're using.